query-builder/revapi.json

```diff
@@ -17,6 +17,28 @@
       }
     },
     "ignore": [
+      {
+        "code": "java.method.addedToInterface",
+        "new": "method <T> com.datastax.oss.driver.api.querybuilder.insert.JsonInsert com.datastax.oss.driver.api.querybuilder.insert.InsertInto::json(T, com.datastax.oss.driver.api.core.type.codec.TypeCodec<T>)",
+        "package": "com.datastax.oss.driver.api.querybuilder.insert",
+        "classQualifiedName": "com.datastax.oss.driver.api.querybuilder.insert.InsertInto",
+        "classSimpleName": "InsertInto",
+        "methodName": "json",
+        "newArchive": "com.datastax.oss:java-driver-query-builder:jar:4.0.0-rc2-SNAPSHOT",
+        "elementKind": "method",
+        "justification": "add missing Inert json() method"
+      },
+      {
+        "code": "java.method.addedToInterface",
+        "new": "method com.datastax.oss.driver.api.querybuilder.insert.JsonInsert com.datastax.oss.driver.api.querybuilder.insert.InsertInto::json(java.lang.Object, com.datastax.oss.driver.api.core.type.codec.registry.CodecRegistry)",
+        "package": "com.datastax.oss.driver.api.querybuilder.insert",
+        "classQualifiedName": "com.datastax.oss.driver.api.querybuilder.insert.InsertInto",
+        "classSimpleName": "InsertInto",
+        "methodName": "json",
+        "newArchive": "com.datastax.oss:java-driver-query-builder:jar:4.0.0-rc2-SNAPSHOT",
+        "elementKind": "method",
+        "justification": "add missing Inert json() method"
+      }
```

Comment:
Revapi includes a lot of information in its suggestions, but in practice not every field is necessary. For this error code, the minimal set is `code, new, newArchive, justification`, all the others can be deduced.
For the sake of keeping this file short and easy to read, I think we should not include them.

Reply:
It turns out that the Revapi plugin has an option to include the minimal necessary fields in suggestions. I've enabled it in 6194b1d9b.